packages/lesmis/src/api/queries/useFavouriteDashboardItem.js

```diff
@@ -0,0 +1,23 @@
+import { stringifyQuery } from '@tupaia/utils';
+import { useQuery } from 'react-query';
+import { get } from '../api';
+import { QUERY_OPTIONS } from './constants';
+
+export const useFavouriteDashboardItem = ({ userId }) => {
```

Comment:
```suggestion
export const useGetFavouriteDashboardItems = ({ userId }) => {
```

Just a rename suggestion to not confusing it with the function to set a dashboard item as favourite. It's a bit confusing cos 'favourite' is both a noun (ie. a favourite dashboard) and a verb (ie. the user chose to favourite the dashboard). So this makes things a bit clearer 👍

Reply:
Love it!